imports/plugins/included/product-variant/containers/gridItemControlsContainer.js

```diff
@@ -1,47 +1,50 @@
-import React, { Component } from "react";
+import React, { PureComponent } from "react";
```

Comment:
Totally agreed @zenweasel.  Adding the word "container" to every file name felt really unnecessary to me when they're already in a `/container` directory.  Also, the base component name (without "container" appended) is how they should be referenced with the new API and you should be able to assume that the final component has that name regardless of whether higher order components are being used.

The only reason I didn't do it across the app is because I already edited almost 300 files.  The PR had to end somewhere.  :)

But yeah, as components get updated to use the new API, my vote is to remove the container from the name and standardize things.